### PR TITLE
Add issue templates

### LINKED ISSUE #5: Issue Templates
We do want to have issue templates for:

- Bug Reports
- Feature Requests
- Other issues we can't classify
- Security Policy (strictly speaking this is not an issue template but it is shown on that page)

We do want to use the new Github Issue templates if possible: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

### PR DESCRIPTION
**Pull request by SchoolGuy**
This adds basic issue templates.

Fixes #5

### CODE REVIEW

**Comment by SchoolGuy** [comment]
The CI fails since it is was unmaintained for quite some time.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```
Comment (tiltingpenguin) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: I would prefer to mark this explicitely as an example

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (tiltingpenguin) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]: Ex is not a good abbreviation for example. I would use "E.g."

**Review by tiltingpenguin — APPROVED** [review]
LGTM

**Pull request merged**